src/coprocessor/statistics/analyze.rs

```diff
@@ -147,6 +147,7 @@ impl<S: Snapshot> AnalyzeContext<S> {
                 }
             }
             if stats_version == ANALYZE_VERSION_V2 {
+                hist.append(&data);
```

Comment:
Why `hist.append(&data);` is in the if condition block? Seems it does not match the original logic.